### ISSUE TITLE
Change the study group name to be more inclusive

### ISSUE THREAD

**Issue by BrunoGrandePhD** [sciprog-sfu/sciprog-sfu.github.io#87]
We can change the name from "Scientific Programming Study Group" to "Scientific and Research Programming Study Group". It's a subtle change, but we want to appear as inclusive as possible, because it better represents what we stand for. The SciProg shorthand and URL still applies.

**Comment by lpix** [comment]
Question: what is your target audience?  I am new to this group but I think it would also be nice to promote coding literacy in the lower level Undergrads (maybe helping to take out the fear/dread of learning to code?). If you stick with the name change, I think beginner coders and non-coders might be intimidated at first-glance by the name.

**Comment by razoumov** [comment]
"Scientific and Research Programming Study Group" is tautology: we should use either Scientific or Research. I can't imagine why undergrads would be scared of either one.

**Comment by lpix** [comment]
I'm just saying that out of experience and from other students' opinions. It drives me nuts how people act surprised or make faces when I say I am interested in programming and Computational Biology.

I like the proposal for the name because you know what to expect from the group, but I also like "SciProg" because it is catchy (the name and a cool looking website are what tipped the scale and made me join the group in the first place :) ). 

Another catchy name I have come across is [Stanford's Bio-X program](https://biox.stanford.edu/).

> Bio-X is Stanford's pioneering interdisciplinary biosciences institute, bringing together biomedical and life science researchers, clinicians, engineers, physicists, and computational scientists to unlock the secrets of the human body. 

Either way, I will be glad to attend and help promote the group, whatever the name is.

**Comment by lpix** [comment]
Another thing you can do is keep the name but brand it as SciProg.

**Comment by BrunoGrandePhD** [comment]
@lpix: Our target audience is basically anyone at SFU, but we also welcome people from off-campus. The following quote is taken from our website. 

> All are welcome to this study group, regardless of scientific research area, affiliation or training level.

To be fair, our focus has been on researchers in the sciences, which reflects the organizers' interests. We want to diversify the lessons to match the diversity of the SciProg community. This is easier said than done, especially for fields that are outside the organizers' expertise. 

@razoumov: I agree that the suggested name is a tautology. 

Maybe it's too ambitious to include other fields like the digital humanities. We already have a lot of diversity within the sciences that we are struggling to accommodate. 

**tl;dr** I think we should officially focus on topics relevant to scientific programmers, but welcome people from the digital humanities if the workshops are relevant to them. We'll stick with the current name and SciProg. We can change the language on the website to clarify this position.

**Comment by lpix** [comment]
@brunogrande I can look into a way of canvassing out the Undergrad population and checking out interests to be more inclusive to them. I don't know how the other SciProg organizers feel about more undergrad oriented topics or events.

**Comment by BrunoGrandePhD** [comment]
I think for the time being, we'll stick with our current name and SciProg shorthand. We'll make an effort to explicitly state to be inclusive of other fields such as the digital humanities.
